README.md

```diff
@@ -76,3 +83,75 @@ attention to:
   Operator](https://github.com/luxas/kubeadm-workshop#deploying-the-prometheus-operator-for-monitoring-services-in-the-cluster)
 - [Setting up the custom metrics adapter and sample
   app](https://github.com/luxas/kubeadm-workshop#deploying-a-custom-metrics-api-server-and-a-sample-app)
+
+FAQs
+----
+
+### Why do my metrics keep jumping between a normal value and a very large number?
+
+You're probably switching between whole numbers (e.g. `10`) and milli-quantities (e.g. `10500m`).
+Worry not!  This is just how Kubernetes represents fractional values.  See the
+[Quantity Values](/docs/walkthrough.md#quantity-values) section of the walkthrough for a bit more
+information.
+
+### Why isn't my metric showing up?
+
+First, check your configuration.  Does it select your metric?  You can find the
+[default configuration](/deploy/custom-metrics-config-map.yaml) in the deploy directory,
+and more information about configuring the adapter in the [docs](/docs/config.md).
```

Comment:
It would be helpful to have a command to return all metrics. Not only the registered metric names but the actual per pod/object metrics. That would help to validate this.

Reply:
I agree. I'm currently struggling with the config and was hoping there's an easy way to list what it can "see".

Reply:
This only shows what metrics it knows about but you still might not have a metric with the right labels for a particular HPA. In that case it would be useful to have a expression that also returns the per-object/pod metrics.

Reply:
Yeah, initially I came to the same conclusion eventually. 😅

Reply:
can you file a separate issue for that?